templates/zerver/api/api-doc-template.md

```diff
@@ -0,0 +1,32 @@
+{generate_api_title(API_ENDPOINT_NAME)}
+
+{generate_api_description(API_ENDPOINT_NAME)}
+
+## Usage examples
+
+{start_tabs}
+{tab|python}
+
+{generate_code_example(python)|API_ENDPOINT_NAME|example(admin_config=True)}
```

Comment:
Why `admin_config=True` here?

Reply:
That was a slight overlook on my part. Fixed.